shotover-proxy/src/transforms/cassandra/sink_cluster.rs

```diff
@@ -356,7 +428,7 @@ pub struct CassandraNode {
     pub address: IpAddr,
     pub _rack: String,
     pub _tokens: Vec<String>,
-    outbound: Option<CassandraConnection>,
+    pub outbound: Option<CassandraConnection>,
```

Comment:
Because if I use `get_connection` then that will implicitly create a new connection which would perform really badly in the new case where I access outbound directly.

Reply:
Doesn't `get_connection` check if it's none before creating a new one?

Reply:
Approved because I'm certainly missing something here.